EncoreTickets.SDK/Venue/VenueServiceApi.cs

```diff
@@ -60,6 +60,22 @@ public Models.Venue GetVenueById(string id)
             return result.Data;
         }
 
+        /// <summary>
+        /// Update the venue by its id.
+        /// </summary>
+        /// <param name="venueId">Venue ID</param>
+        /// <param name="venue">Venue for update</param>
+        /// <returns>Updated venue</returns>
+        public Models.Venue UpdateVenueById(string venueId, Models.Venue venue)
```

Comment:
I think this method should just be UpdateVenue and not accept the 'venueId' param.  Instead the path in the url should source the id from the venue param.